Core/Core/Profile/Settings/APIPairingCode.swift

```diff
@@ -61,22 +63,52 @@ public struct PostAccountUserRequest: APIRequestable {
         )
     }
 
-    public struct Body: Encodable, Equatable {
-        public struct Pseudonym: Encodable, Equatable {
+    public struct Body: Codable, Equatable {
+        public struct Pseudonym: Codable, Equatable {
             let unique_id: String
             let password: String
         }
-        public struct PairingCode: Encodable, Equatable {
+        public struct PairingCode: Codable, Equatable {
             let code: String
         }
-        public struct User: Encodable, Equatable {
+        public struct User: Codable, Equatable {
             let name: String
             let initial_enrollment_type: String
+            let terms_of_use: Bool = true
         }
         let pseudonym: Pseudonym
         let pairing_code: PairingCode
         let user: User
     }
+
+    public func urlRequest(relativeTo baseURL: URL, accessToken: String?, actAsUserID: String?) throws -> URLRequest {
```

Comment:
Implementing this method so we can use a custom `baseURL` for creating acccount that may different than what env.api is set to.

Reply:
I don't think this is necessary to get the desired effect. You can have an absolute url in the `path`, which allows you to point to a different domain, and have an explicit null authorization in `headers` to exclude sending the token. `GetMobileVerifyRequest` does this.

Reply:
What are the advantages of this over using an absolute url with the usual setup? (it looks like this does a lot, so there may well be something different)

Reply:
This worked well, thanks for the suggestion.